src/lib/hooks/hasConnectionChange.js

```diff
@@ -12,15 +12,13 @@ const log = logger.child({ from: 'hasConnectionChange' })
 export const useConnection = () => {
```

Comment:
i've read `@react-native-community/netinfo` docs and it's seems all you need in `useConnection` is:

```suggestion
export const useConnection = () => {
  const { isConnected = false } = useNetInfo() || {}
  
  return isConnected
})
```